modules/auxiliary/admin/mssql/mssql_escalate_dbowner.rb

```diff
@@ -17,19 +18,25 @@ def initialize(info = {})
       },
       'Author'         => [ 'nullbind <scott.sutherland[at]netspi.com>'],
       'License'        => MSF_LICENSE,
-      'References'     => [[ 'URL','http://technet.microsoft.com/en-us/library/ms188676(v=sql.105).aspx']]
+      'References'     => [[ 'URL','http://technet.microsoft.com/en-us/library/ms188676(v=sql.105).aspx']],
+      'SessionTypes'   => %w[MSSQL]
     ))
   end
 
   def run
     # Check connection and issue initial query
-    print_status("Attempting to connect to the database server at #{rhost}:#{rport} as #{datastore['USERNAME']}...")
-    if mssql_login_datastore
-      print_good('Connected.')
+    if session
+      set_session(session.client)
+      print_good("Using session #{datastore['SESSION']}.")
```

Comment:
Looks like we have access to `session` here as it's being handled to for the logic on the line above, so could we just call: 
```suggestion
      print_status("Using existing session #{session.sid}")
```

Thoughts on maybe a `print_status` here rather than `print_good`?

Reply:
Seems this is in several places, so if we go ahead with this, it'll need updated in each place 👍

Reply:
Per your previous comment, I just stuck that in `set_session` 👍